docs/project/list-of-diagnostics.md

```diff
@@ -28,6 +28,7 @@ Currently the identifiers `SYSLIB0001` through `SYSLIB0999` are carved out for o
 |  __`SYSLIB0013`__ | Uri.EscapeUriString can corrupt the Uri string in some cases. Consider using Uri.EscapeDataString for query string components instead. |
 |  __`SYSLIB0015`__ | DisablePrivateReflectionAttribute has no effect in .NET 6.0+ applications. |
 |  __`SYSLIB0016`__ | Use the Graphics.GetContextInfo overloads that accept arguments for better performance and fewer allocations. |
+|  __`SYSLIB0017`__ | Strong name signing is not supported on this platform.|
```

Comment:
We can probably omit the "on this platform" portion of the message. Similar to the others, we might want to say the API throws `PlatformNotSupportedException`.

Reply:
The message copied from the resource string passed in PNSE constructor, i can update